### PR TITLE
bots: Fix URL for rhel-8.0 repo

### PR DESCRIPTION
**Pull request by marusak**
* [x] image-refresh rhel-8-0

### CODE REVIEW

**Comment by cockpituous** [comment]
image-refresh in progress on 1-ci-srv-05.
Log: https://209.132.184.41:8493/logs/image-refresh-12638-20190826-124040/

**Comment by cockpituous** [comment]
image-refresh in progress on 3-cockpit-7.
Log: https://209.132.184.41:8493/logs/image-refresh-12638-20190826-124040/

**Comment by cockpituous** [comment]
Task failed: https://209.132.184.41:8493/logs/image-refresh-12638-20190826-124040/

**Comment by cockpituous** [comment]
image-refresh in progress on 3-ci-srv-04.
Log: https://209.132.184.41:8493/logs/image-refresh-12638-20190826-131717/

**Comment by cockpituous** [comment]
image-refresh in progress on 2-cockpit-10.
Log: https://209.132.184.41:8493/logs/image-refresh-12638-20190826-131717/

**Comment by cockpituous** [comment]
image-refresh rhel-8-0 done: https://github.com/cockpituous/cockpit/commits/image-refresh-rhel-8-0-20190826-134040

**Comment by marusak** [comment]
Ehm, now I think if we wanted to move from `rhel-8.0.0 -> rhel-8.0.1` or stay at `8.0.0` in which case url should change to something like `http://download.redhat.com/rhel-8/rel-eng/RHEL-8/latest-RHEL-8.0.0/compose/BaseOS/x86_64/`.

File: bots/images/scripts/rhel.setup [hunk 1]
```diff
@@ -97,7 +97,7 @@ EOF
 
     elif [ "${IMAGE#rhel-8*}" != "$IMAGE" ]; then
         case "$IMAGE" in
-            rhel-8-0) REPO="latest-RHEL-8.0" ;;
+            rhel-8-0) REPO="latest-RHEL-8.0.1" ;;
```
Comment (martinpitt) [inline bots/images/scripts/rhel.setup]: My intent was that "latest-RHEL-8.0" would always point to the nightlies for the upcoming 8.0.z, and that we *don't* want to hardcode the .1. If that is not what it is, and there is no such link, then fine.

**Comment by martinpitt** [comment]
IMHO it makes sense to always follow the latest z-stream updates. There is little point on always testing on 8.0.0, especially at this point.

**Comment by marusak** [comment]
So I guess this PR is good to go?

**Comment by martinpitt** [comment]
> So I guess this PR is good to go?

 *if* the current URL is really "8.0.0", then yes. If it's "8.0.z latest", then let's keep that.

**Pull request merged**